pagopa/io-app
# chore: [IA-146] Show a thank-you screen at the end of the onboarding

## Short description
This PR adds a thank-you screen at the end of the onboarding.

### Please note that english translation is not completed.

![Simulator Screen Shot - iPhone 12 - 2022-03-11 at 18 58 21](https://user-images.githubusercontent.com/467098/157923943-13a9d94b-0f08-48f4-9480-3ce3d3fb10aa.png)

## List of changes proposed in this pull request
- Add a `completeOnboardingSaga` that is called at the end of the onboarding during the startup
- The saga navigates to `OnboardingCompletedScreen` and waits for the `completeOnboarding` action to be dispatched

## How to test
Delete the app from the device and run it from scratch. At the end of the first onboarding a thank you page should be presented to the user.

https://user-images.githubusercontent.com/467098/157922171-d24dfa4f-3313-43d9-a2cd-0d91c58de1b5.mp4
